bot/exts/evergreen/timed.py

```diff
@@ -21,6 +21,8 @@ async def timed(self, ctx: commands.Context, *, command: str) -> None:
         """Time the command execution of a command."""
         new_ctx = await self.create_execution_context(ctx, command)
 
+        ctx.subcontext = new_ctx
+
         if not new_ctx.command:
```

Comment:
Why don't we do this?
```suggestion
        if not ctx.subcontext.command:
```